logger/logger.swift

```diff
@@ -1,10 +1,188 @@
 //
 //  logger.swift
-//  logger
+//  Test
 //
-//  Created by Daniel Brain on 10/12/20.
+//  Created by Daniel Brain on 10/11/20.
 //
 
-class logger {
+import Foundation
 
+let DEFAULT_PAYLOAD: Payload = [:]
+
+class Logger {
+    private let url: URL
+    private let prefix: String?
+    private let logLevel: LogLevel
+    private let flushInterval: Double
+    
+    private var events = Events()
+    private var headerBuilders: [() -> Headers] = []
+    private var payloadBuilders: [() -> Payload] = []
+    private var trackingBuilders: [() -> Payload] = []
+    
+    init(
+        url : String,
+        prefix : String? = nil,
```

Comment:
`prefix: String? = nil` => `identifier: String = Bundle.main.bundleIdentifier`, thoughts?

Reply:
Aha! Didn't know about `Bundle.main.bundleIdentifier`. Is this the identifier for whatever app includes this library?

Reply:
Yes that's right, so a unique way to identify application :-)